packages/components/tag/docs/additional-info.md

```diff
@@ -0,0 +1,23 @@
+# TagList
+
+## Description
+
+A `TagList` component is used as wrapper for the list of tags. It has predefined spacings and flex display so that tags are grouped next to each other.
+
+## Usage
+
+The `<TagList>` component accepts `<Tag>` element as `children` props.
+Here's the example how to use them:
+
+```jsx
+<TagList>
+  {examples.map((tag, index) => (
+    <Tag
+      key={index}
+    >
+      {tag}
+    </Tag>
+  ))}
+</TagList>
+/>
```

Comment:
My suggestion would be to use a bit more tangible example (so that everyone knows it should be an array of `ReactNode`s):
```suggestion
<TagList>
  {["apple", "orange"].map((tag, index) => (
    <Tag
      key={index}
    >
      {tag}
    </Tag>
  ))}
</TagList>
```